cmd/security-agent/app/runtime.go

```diff
@@ -18,10 +22,67 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/logs/pipeline"
 	"github.com/DataDog/datadog-agent/pkg/logs/restart"
 	secagent "github.com/DataDog/datadog-agent/pkg/security/agent"
+	secconfig "github.com/DataDog/datadog-agent/pkg/security/config"
+	"github.com/DataDog/datadog-agent/pkg/security/policy"
+	sprobe "github.com/DataDog/datadog-agent/pkg/security/probe"
+	"github.com/DataDog/datadog-agent/pkg/security/rules"
 	"github.com/DataDog/datadog-agent/pkg/status/health"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+var (
+	runtimeCmd = &cobra.Command{
+		Use:   "runtime",
+		Short: "Runtime Agent utility commands",
+	}
+
+	checkPoliciesCmd = &cobra.Command{
+		Use:   "check-policies",
+		Short: "Check policies and return a report",
+		RunE:  checkPolicies,
+	}
+
+	checkPoliciesArgs = struct {
+		dir string
+	}{}
+)
+
+func init() {
+	runtimeCmd.AddCommand(checkPoliciesCmd)
+	checkPoliciesCmd.Flags().StringVar(&checkPoliciesArgs.dir, "policies-dir", "/etc/datadog-agent/runtime-security.d", "Path to policies directory")
```

Comment:
This is a reasonable default but should it instead use the one specified in agent config (yaml/env vars) as default?